packages/landing/static/examples/index.openlayers.attribution.wmts.3857.html

```diff
@@ -9,7 +9,7 @@
         type="text/css"
     />
     <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/build/ol.js"></script>
-    <script src="../lib/attribution.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@basemaps/attribution/dist/attribution.js"></script>
```

Comment:
We really should add SRI to this, and pin it to a version, I guess its chicken/egg though since we need it published first.